CodePush.js

```diff
@@ -436,6 +477,10 @@ if (NativeCodePush) {
       DOWNLOADING_PACKAGE: 7,
       INSTALLING_UPDATE: 8
     },
+    SyncMode: {
```

Comment:
How about `SyncTrigger`? The understanding is we are triggering `sync` during an event, `ON_APP_START`, `ON_APP_RESUME` or `MANUAL` (when the user calls `sync` manually).